swim/update_rollup_test.go

```diff
@@ -99,11 +100,23 @@ func (s *RollupTestSuite) TestFlushesAfterInterval() {
 	})
 
 	s.NotNil(s.r.FlushTimer(), "expected timer to be set")
+
+	// This is a race condition which never got triggered yet. If the scheduler
```

Comment:
Can you induce this race condition? Is it practically possible to induce this? If not, I'd say there is no race condition and perhaps the comment is slightly confusing. If there is a race condition, we should file a task and fix it.

Also for clarity, it might help to prefix the comment with "TODO:" as my first impression was that the commented line was actually fixing it or something.